src/Umbraco.Web.UI.Client/src/packages/documents/documents/workspace/document-workspace.context.ts

```diff
@@ -434,6 +415,9 @@ export class UmbDocumentWorkspaceContext
 		this.readOnlyGuard?.addRule({
 			unique: identifier,
 			message,
+			/* This guard is a bit backwards. The rule is permitted to be read-only. 
+			If the user does not have permission, we set it to true = permitted to be read-only. */
+			permitted: true,
```

Comment:
From an outsider trying to use the readOnly guard, I only stumbled across this PR and realised I had to set permitted to true as opposed to false like the `propertyWriteGuard`

Would it best if this could be aligned like the others, just to avoid confusion.
And another comment is that the docs page mentions not to use readonlyGuard. Is this the case or can us package devs use it ?

Reply:
We already relied on it in core, and to keep this fix as simple as possible, I just corrected the code. We want to move away from it in core as well, and the current plan is for it to be deprecated and removed eventually. If you can solve your case with the view/write guards, I recommend using them. If you can't, please let us know.

As mentioned in the comment, the DX is a bit backward compared to the view/write guards. We want guards to grant access to something instead of removing it.